### PR TITLE
fix(curriculum): require meta charset for  cat photo project

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

contribute to #46540

image

### CODE REVIEW

File: curriculum/challenges/_meta/learn-html-by-building-a-cat-photo-app/meta.json [hunk 1]
```diff
@@ -277,5 +277,10 @@
...
       "5ef9b03c81a63668521804ee",
       "Step 67"
+    ],
+    [
+      "62bb4009e3458a128ff57d5d",
+      "Step 68"
     ]
-  ]}
+  ]
+}
```
Comment (jeremylt) [inline curriculum/challenges/_meta/learn-html-by-building-a-cat-photo-app/meta.json:286]: What's the convention about this in the fCC codebase? Most files seem to have that extra newline at the end

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804ee.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ dashedName: step-67
 
 All pages should begin with `<!DOCTYPE html>`. This special string is known as a <dfn>declaration</dfn> and ensures the browser tries to meet industry-wide specifications.
 
-To complete this project, add this declaration as the first line of the code.
+add this declaration as the first line of the code.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804ee.md:12]: Capitalize the A in Add? Should this be part of the previous paragraph?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
+---
+id: 62bb4009e3458a128ff57d5d
+title: Step 68
+challengeType: 0
+dashedName: step-68
+---
+
+# --description--
+
+Oh, we almost forgot. We should allow people to use their own native language. You can set the browser to encode multiple languages, by adding `meta` tag as a child for `head`, and set its `charset` attribute to.
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:10]:
```suggestion
One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to:
```

**Review by moT01 — COMMENTED** [review]
Minor grammar suggestions - we should avoid using `we` if we can, as described in [the docs.](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions)

**Review by moT01 — DISMISSED** [review]
LGTM 🎉

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+title: Step 68
+challengeType: 0
+dashedName: step-68
+---
+
+# --description--
+
+One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to:
+
+`UTF-8`
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:12]:
```suggestion
One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```

This seems weird on a separate line.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md (hunk 4 above)
Comment (Sboonny) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:12]: It's but we had issues with code block at the end of the sentence.

bbsmooth recommend new line with the value in it, to support mobile users accessibility.

Comment (moT01) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:12]: I don't like it on its own line either - and would vote to put it back on the same line. For some context, we had [some discussion on this](https://github.com/freeCodeCamp/freeCodeCamp/issues/46052) and made some changes already. The conclusion was to do this for the first few steps of this project.

**Review by naomi-lgbt — APPROVED** [review]
Should be good to go now

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by moT01 — APPROVED** [review]

**Pull request merged**